tests/usermetas.test.js

```diff
@@ -95,7 +95,7 @@ describe('usermetas', async () => {
       const next = function(err) {
         try {
           assert(err == undefined, 'there should not be an error');
-          td.verify(res.json(response));
```

Comment:
This could have been kept, the fact that the response from `updateDynamicUserMeta` has been sent in JSON is part of the specification for this middleware.